### PR TITLE
(issue 874): implemented Axis.min() and Axis.max() methods

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

**Review by gdementen — COMMENTED** [review]
I would add either an explanation **or** an example that it fails on non-numeric labels.

**Comment by gdementen** [comment]
what's blocking this PR (except failed tests?)

File: larray/core/axis.py [hunk 1]
```diff
@@ -614,6 +614,60 @@ def equals(self, other):
...
+        --------
+        Fails on non-numeric labels.
+
+        Examples
+        --------
+        >>> time = Axis('time=1991..2020')
+        >>> time.min()
+        1991
+
+        >>> country = Axis('country=Belgium,France,Germany')
```
Comment (gdementen) [inline larray/core/axis.py:636]: isn't having the Warning AND a failed example a bit overkill? Isn't the warning enough by itself?

File: larray/core/axis.py [hunk 2]
```diff
@@ -614,6 +614,60 @@ def equals(self, other):
...
+        --------
+        Fails on non-numeric labels.
+
+        Examples
+        --------
+        >>> time = Axis('time=1991..2020')
+        >>> time.max()
+        2020
+
+        >>> country = Axis('country=Belgium,France,Germany')
```
Comment (gdementen) [inline larray/core/axis.py:663]: idem

File: larray/core/axis.py (hunk 1 above)
Comment (alixdamman) [reply larray/core/axis.py:636]: Depends on what users actually read. Some of them may only read the examples. 
It doesn't hurt to have both IMO.

**Comment by gdementen** [comment]
LGTM then

**Pull request merged**